include/zrepl_mgmt.h

```diff
@@ -63,6 +64,7 @@ typedef struct zvol_info_s {
 	int		is_io_ack_sender_created;
 	uint64_t	checkpointed_io_seq;
 	taskq_t		*uzfs_zvol_taskq;	/* Taskq for minor management */
+	uint16_t 	rebuild_cnt;		/* Rebuild_cnt */
```

Comment:
Why is this not part of zvol_rebuild_t? either remove the first or the latter IMHO

Reply:
Moved it to rebuild related info zvol_state_t struct.